naughty/debian-stable/1372-libvirt-crashes-on-test-teardown

```diff
@@ -0,0 +1,9 @@
+*testAddDiskNFS*
+*
+Process * (libvirtd) of user * dumped core.
+*
+#0  * __GI___pthread_mutex_lock (libpthread.so.0)
+#1  * virQEMUDriverGetConfig (libvirt_driver_qemu.so)
+#2  * n/a (libvirt_driver_qemu.so)
```

Comment:
I suggest to skip lines #0 and #2, as there might be too much fuzz. Just matching on #1 and #3 should be enough.

Reply:
It was actually skipped as expected - see https://logs.cockpit-project.org/logs/pull-1373-20201104-165549-f6b48ae6-debian-stable-cockpit-project-cockpit/log.html

Shall I still change it or keep it as specific as it get?

Reply:
HA testInlineConsoleWithUrlRoot fails with the exact same way - I think i will need to remove the test pattern match.